addins/Cake.AutoRest.yml

```diff
@@ -2,9 +2,10 @@ Name: Cake.AutoRest
 NuGet: Cake.AutoRest
 Assemblies:
 - "/**/Cake.AutoRest.dll"
-Repository: https://github.com/agc93/Cake.AutoRest
+Repository: https://github.com/agc93/Cake.AutoRest/
 Author: Alistair Chapman
 Description: "Cake addin for generating client code from compatible API definitions (including Swagger)."
+Prerelease: "true"
```

Comment:
@Jericho Wonder if this is correct to switch to pre-release just because an newer pre-release exists?

Reply:
As mentioned previously, I was in favor of ignoring pre-release addins but @gep13 thought otherwise.